packages/account/src/Styles/account.scss

```diff
@@ -1880,7 +1879,6 @@ $MIN_HEIGHT_FLOATING: calc(
         }
         &__content {
             @include typeface(--paragraph-left-normal-prominent, none);
-            color: var(--text-general);
             line-height: 1.43;
```

Comment:
Should we keep this?

Reply:
Yes because there are other elements using this class

Reply:
Yes we need 'line-height', it’s a custom value for this class, we don’t have it in variables

Reply:
It's safe to use `1.4` instead of `1.43` per our discussion with @sara-fs and @zaki-binary

Reply:
I've removed this class from the Text component here and I reset its styles to defaults for other elements.